Mathlib/Analysis/NormedSpace/OperatorNorm/Prod.lean

```diff
@@ -139,3 +151,37 @@ end Prod
 end ContinuousLinearMap
 
 end SemiNormed
+
+section Normed
+
+namespace ContinuousLinearMap
+
+section FirstSecond
+
+variable (𝕜 E F)
+
+/-- The operator norm of the first projection `E × F → E` is exactly 1 if `E` is nontrivial. -/
+@[simp] lemma norm_fst [NormedAddCommGroup E] [NormedSpace 𝕜 E]
```

Comment:
I'm pretty sure no such typeclass exists, since I copied the setup from `ContinuousLinearMap.norm_id_le` / `norm_id` which is probably the most obvious place such a typeclass would be used if it existed.